src/main/resources/db/changelog/changes/11-create-shoppingcarts-table.yaml

```diff
@@ -0,0 +1,28 @@
+databaseChangeLog:
+  - changeSet:
+      id: create-shopping_carts-table
+      author: alex_liubchenko
+      changes:
+        - createTable:
+            tableName: shopping_carts
+            columns:
+              - column:
+                  name: id
+                  type: bigint
+                  autoIncrement: true
+                  constraints:
+                    primaryKey: true
+                    nullable: false
```

Comment:
unnecessary make nullable constraint for primaryKey
primaryKey has this (nullable) constraint by default